src/plugins/wb5-AJAX-submit/wb5-AJAX-submit.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @title WET-BOEW wb5-AJAX-submit
+ * @overview This plugin implements AJAX request for form data
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @masterbee, @namjohn920
+ **/
+
+( function( $, wb ) {
+"use strict";
+
+var $document = wb.doc,
+	componentName = "AJAX-submit",
```

Comment:
The component name need to start with "wb-". Probably name it like: "wb-submit-ajax"